ci/debian/centreon-connector-perl.install

```diff
@@ -1 +1 @@
-debian/tmp-centreon-collect/usr/lib64/centreon-connector/centreon_connector_perl usr/lib/centreon-connector
+debian/tmp-centreon-collect/usr/bin/centreon_connector_perl usr/bin
```

Comment:
why ?

Reply:
right location after build and destination like rpm package (/usr/bin)

Reply:
on rpm package it's /usr/lib64/centreon-connector/

Reply:
if you change this location, we also have to update centengine configuration, particularly the default configuration of engine. Did you do that already? If not, I think it is better to keep this path even if it is not standard.